app/src/main/java/com/example/hwone/utils/ValidationUtils.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.hwone.utils
+
+object ValidationUtils {
+
+    fun isValidEmail(email: String): Boolean {
+        return android.util.Patterns.EMAIL_ADDRESS.matcher(email).matches()
+    }
+
+    fun isValidPassword(password: String): Boolean {
+        val passwordPattern = "^(?=.*[0-9])(?=.*[A-Z]).{8,}$"
```

Comment:
This pattern does not match to "Your password must include a minimum of 8 characters" error description. Or vice versa.

Reply:
Done. Now the password can contain a minimum of 8 characters. But the task stated that we can come up with our own logic :)

Reply:
No, I mean, when I input "12345678" in password field, I have gotten "Your password must include a minimum of 8 characters" error description, but password clearly has 8 characters.